app/components/frame-viewer.jsx

```diff
@@ -104,6 +104,21 @@ export default class FrameViewer extends React.Component {
           </FrameWrapper>
         </PanZoom>
       );
+    } else if (this.props.isGroupSubject && this.props.linkToSubject) {
+      return (
+        <a href={this.props.linkToSubject} style={{ display: 'block', border: '1px solid cyan'}}>
```

Comment:
Using `a` here means the whole page gets reloaded, and PFE gets run again from scratch. This should use client-side routing instead.

Reply:
This link also has no link text, so there’s no indication as to what clicking it does. Assistive technology might fall back to using the `src` of the wrapped image.

Reply:
In Firefox, you can right-click and inspect the accessibility API for the link. It has no label and I think is falling back to the target URL as the value.
![Accessibility inspector for a subject image in Firefox, showing no accessible label.](https://user-images.githubusercontent.com/59547/125613325-13e326d3-e031-4c8f-a66d-fb79908b8188.png)

Reply:
🆗 on adding a text label for accessibility. The solution I'm implementing is adding attributes like `title="Subject 1234"`.

❓ on changing `<a>` to `<Link>`, though. Sorry, I should have documented this here:
- a `<Link>` in the Subject Talk page that links to another Subject Talk page (within the same project) doesn't seem to work correctly (Chrome 91, FF 89)
- Clicking on the link once will change the URL on the browser's navigation bar, but the page won't refresh. 
  - (Clicking several times, on different links, results in different shenanigans based on the router history.)
- This behaviour was noted in the other branch as well: https://github.com/zooniverse/Panoptes-Front-End/pull/5984#issuecomment-878633203
- I've done some additional reading on this and the "correct solution" is to wrap the... Subject Talk page?... with `withRouter` but this was a rabbit hole. https://stackoverflow.com/questions/44823590/react-router-4-doesnt-update-ui-when-clicking-link

Reply:
I'm going to try `withRouter` as a solution tomorrow, but if that doesn't work out, I'll need to fall back on using `<a>` for this stage of the project.

Reply:
Oh that’s weird. The component is definitely set up to fetch a new subject when the ID in the URL changes.

Reply:
The subject page is still using the deprecated `componentWillReceiveProps` so it might be worth upgrading it to `componentDidUpdate`.

Reply:
Wait, no, the issue is with `SubjectPageContainer`'s `setSubject()` using `this.props`, when it might actually need `nextProps`. Classic React shenanigans!

Reply:
HAH solved. `<a>` is now replaced with `<Link>` 👍

Reply:
D'oh! That mistake with the props might explain some other Talk bugs that have come up over the years. Could you change the function from `componentWillReceiveProps(nextProps)` to `componentDidUpdate(prevProps)`? That seems like a small change to me, and `componentWillReceiveProps` isn't part of React any more. Removed in React 15 somewhere, I think, because it's unpredictable if it fires multiple times during a render cycle.